lib/rss/xmlscanner.rb

```diff
@@ -111,11 +111,7 @@ def on_stag_end_empty(name)
     private
     def entity(ref)
       ent = ENTITIES[ref]
-      if ent
-        ent
-      else
-        wellformed_error("undefined entity: #{ref}")
-      end
+      ent || wellformed_error("undefined entity: #{ref}")
```

Comment:
```suggestion
      if ent
        ent
      else
        wellformed_error("undefined entity: #{ref}")
      end
```

Not in `Style/StringLiterals`.